### PR TITLE
CORE-5143: Implemented filter for group retrieval

### PR DESCRIPTION
**Pull request by zilladotexe**

### CODE REVIEW

File: src/main/java/com/boomi/flow/services/okta/authorization/AuthorizationManager.java [hunk 1]
```diff
@@ -150,6 +153,32 @@ public ObjectDataResponse groups(ObjectDataRequest request) {
         );
     }
 
+    private String buildFilterStringFromRequest(ObjectDataRequest objectDataRequest, String name, String propertyName){
+
+        String filter = "";
+        
+        if (objectDataRequest.getObjectData() != null && objectDataRequest.getObjectData().size() > 0) {
+            for (MObject requestedGroups : objectDataRequest.getObjectData()) {
```
Comment (josecollazzi) [inline src/main/java/com/boomi/flow/services/okta/authorization/AuthorizationManager.java]: @domholmes could you change the restedGroupds to singular.

**Comment by josecollazzi** [comment]
@domholmes just a minor change. I have also found that the sdk library is a bit old, have you try with a new version?

**Review by josecollazzi — CHANGES_REQUESTED** [review]

File: src/main/java/com/boomi/flow/services/okta/authorization/AuthorizationManager.java (hunk 1 above)
Comment (zilladotexe) [reply src/main/java/com/boomi/flow/services/okta/authorization/AuthorizationManager.java]: Done

**Comment by zilladotexe** [comment]
> @domholmes just a minor change. I have also found that the sdk library is a bit old, have you try with a new version?

Done.

**Review by josecollazzi — APPROVED** [review]

**Pull request merged**